docs/admin/deployment/onboarding_admins.rst

```diff
@@ -31,60 +31,49 @@ To set up AW2, follow these steps:
    and `set an admin password on the welcome screen <https://tails.boum.org/doc/first_steps/welcome_screen/administration_password/>`__
 2. Ensure that Tails and the SecureDrop version on AW1 are up-to-date.
    If not, update now by following the :ref:`most recent upgrade guide <latest_upgrade_guide>`.
-3. Insert the empty AW2 USB drive.
-4. Launch the Tails installer (**Applications ▸ Tails ▸ Tails Installer**) and install Tails on AW2.
-   This will delete all data on the AW2 USB drive.
-5. Shut down AW1.
-6. Boot AW2.
-7. Configure its persistent volume (**Applications ▸ Tails ▸ Configure persistent volume**).
-   Set a unique passphrase for AW2 and record it securely. Enable all persistence options.
-8. Reboot AW2, unlock its persistent volume, and set an admin password on the welcome screen.
-9. Open the file manager (**Applications ▸ Accessories ▸ Files**).
-10. Insert AW1. It should show up in the list of storage devices in the file manager under
-    a label like "7.0 GB Encrypted". Click the label and enter the drive
-    password when prompted to unlock it.
-11. In a terminal, type the following command:
-
-    ``rsync -a /media/amnesia/TailsData/Persistent/securedrop ~/Persistent``
+3. Log into the *Journalist Interface* using your admin credentials, and create
+    a new user account with admin rights. Record its passphrase securely;
+    you will add it to the password manager on AW2.
 
-    This will copy *only* the ``securedrop`` directory from AW1 to AW2.
+    (You will need to on-board the new admin's 2FA device to complete this step.
+    If this is not possible yet, you can defer it until later.)
+4. Insert the empty AW2 USB drive.
+5. Launch the Tails Cloner (**Applications ▸ Tails ▸ Tails Cloner**).
+   Select the option to **Clone the current Tails.** This will delete all data on the AW2 USB drive.
+6. Check the box marked **Clone the current Persistent Storage.**
+7. Click **Install**.
+8. Choose a unique passphrase for AW2 and record it securely.
+9. Shut down AW1.
+10. Boot AW2, set an Admin password on the welcome screen, unlock the Persistent Storage, and
+    enable all the options in the Persistent Storage settings.
+11. Record the new passphrase for AW2 securely.
 12. Generate a new keypair on AW2 using the following command:
```

Comment:
Does this overwrite the existing keypair from AW1? It seems like a smoother workflow might be:

1. Move AW1 SSH keypair somewhere else
2. Generate new keypair
3. Add AW1 keypair to ssh-agent, run ssh-copy-id
4. Reset ssh-agent + delete AW1 keypair
5. Verify ssh works with `ssh app`, etc.

What do you think?

Reply:
@legoktm I just stepped through this entire process again from scratch (it had been a while), and confirmed that it does *not* overwrite the existing keypair from AW1. Interestingly, it also remembered the Admin PW, and was essentially a perfect copy of AW1. I booted up AW2 for the first time, unlocked persistent storage, the Admin PW was already enabled, and as soon as I logged in I was able to run `ssh app` and access my server.

Unless there's a reason for us to deliberately delete the keypair and generate a fresh keypair on AW2, I think these steps are (mostly) still correct. I'll tack on a commit to make them completely correct, but let me know if we want to do the SSH keypair regeneration.

Reply:
@nathandyer, is it that 

https://github.com/freedomofpress/securedrop-docs/blob/5cd6ae2fe228d53b10ffd91298b43810ea3bcfac/docs/admin/deployment/onboarding_admins.rst?plain=1#L52

doesn't overwrite original keypair 1 from AW1? or is it that in

https://github.com/freedomofpress/securedrop-docs/blob/5cd6ae2fe228d53b10ffd91298b43810ea3bcfac/docs/admin/deployment/onboarding_admins.rst?plain=1#L63-L64

we load first the new keypair 2 on AW2 and then the original keypair 1 from AW1?

The latter was my intention in #484 and is still what I understand the current instructions to do.

Reply:
Line 52 does indeed overwrite the original keypair (it prompts if you intend to overwrite, and all you have to do is type Y to proceed).

Your understanding is correct, I'll modify these steps to match that intended behavior.